packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -81,6 +127,14 @@ const Collapse = React.forwardRef(function Collapse(props, ref) {
     TransitionComponent = Transition,
     ...other
   } = props;
+
+  const styleProps = {
+    ...other,
```

Comment:
```suggestion
    ...props,
```

I overlooked this, sorry :)